MIT.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
+
```

Comment:
- [ ] [OPTIONAL] Try to personalize this MIT.md file, with adding the year and full name information 👍